lib/rake/application.rb

```diff
@@ -84,6 +84,11 @@ def init(app_name='rake')
       standard_exception_handling do
         @name = app_name
         args = handle_options
+        if args.size > 1 && args.last =~ /.*\]/
+          clean_args = ""
```

Comment:
This could just be 
```rb
if args.size > 1 && args.last =~ /.*\]/
  args = [args.join]
end
```

right?


Also if the only thing that's stopping this from going out is more tests, I'd be happy to write them.